AmazonWeb1/src/main/java/com/amazon/service/ProductService.java

```diff
@@ -66,7 +66,7 @@ static ProductService getInstance() {
      * @param productId product id of the product object
      * @return Represent {@link Product} in product list
      */
-    Product get(final long productId);
+    Product get(final Long productId);
```

Comment:
Why are you changing it to wrapper?

Reply:
Collection framework uses wrapper class objects.